app/views/layouts/application.html.erb

```diff
@@ -110,7 +110,7 @@
                       <%= link_to "Protocols", admin_protocols_path, class: "dropdown-item #{active_controller?("admin/protocols")}" %>
                     <% end %>
 
-                    <% if current_admin.accessible_admins(:manage).any? %>
+                    <% if current_admin.accessible_facilities(:manage).any? %>
```

Comment:
We're using `current_admin.accessible_facilities(:manage).any?` as a proxy for a lot of checks. Wondering if we can do any better.

Reply:
Yeah, we should not call these so many times, for one, we should wrap all of them in a single block.

Separately, this one actually should be `accessible_admins` – it's more correct. But I ended up using `accessible_facilities` for speed, since the other one is slower and will have an impact on all pages. But it's a good enough proxy for the admin check for now.

Reply:
I'll drop a comment about this.